src/components/Search/Search.scss

```diff
@@ -140,21 +159,18 @@
     .wkp-search-toggle-button {
       margin-right: 75px;
     }
-
-    .wkp-search-toggle-container--open {
-      margin: 0;
-    }
   }
 
   &.tm-w-xs,
   &.tm-w-s {
     .wkp-search-button {
-      height: 39px;
+      width: 40px;
     }
 
     .react-autosuggest__input {
+      width: calc(100% - 90px);
```

Comment:
If you use display flex the width should be automatic calculated.

Reply:
The problem here is that the clear button is inserted dynamically and with the larger text font the container content will overflow once there is an input value